bluepyemodel/access_point/forge_access_point.py

```diff
@@ -1,5 +1,21 @@
 """NexusForgeAccessPoint class."""
 
+"""
+Copyright 2023-2024 Blue Brain Project / EPFL
```

Comment:
Since we added these in 2024, I think technically We should only have `Copyright 2024 Blue Brain Project / EPFL` for those files.